src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import "./Footer.scss"
+
+class Footer extends Component {
+  render() {
+    return (
+      <div className="Footer">
+        <div className="teaSocietyBanner">
+          <video autoplay="true" muted playsinline loop type="video/webm" src="https://cdn.intelligencebank.com/au/share/NOrD/e3oNW/vvAy3/webm/FA+animated+logo_1583847446"></video>
+          <div className="textBox">
+            <div className="textTitle">Tea Society</div>
+            <div className="textDescription">The T2 Tea Society is home to our community of passionate like-minded sippers. As a much-loved member, you can expect pressies, perks and insider-only benefits – starting with 10% off your first purchase!</div>
+          </div>
+          <button>Sign up & take 10% off your first order</button>
+        </div>
+        <div className="bCorpBanner">
+          <div className="textBox">
+            <div className="text">Your favourite tea is now B Corp certified. We are now proudly part of a global community of over 3,000 companies doing business that puts people and place first.
+              </div>
+            <button>Our B Corp accreditation</button>
+          </div>
+        </div>
+        <footer>
+          <div className="leftFooter">
+          <ul>
+            <li><a href="#"><span>Tea Society</span></a></li>
```

Comment:
```suggestion
            <li>
              <a href="#">
                <span>Tea Society</span>
              </a>
            </li>
```
부모 자식 관계가 잘 드러나도록 펼쳐서 작성해주세요. 이런 스타일의 코드 전부!

Reply:
수정했습니다.